src/api/db/structure.sql

```diff
@@ -532,29 +532,29 @@ CREATE TABLE `groups` (
   `parent_id` int(11) DEFAULT NULL,
   `email` varchar(255) COLLATE utf8_bin DEFAULT NULL,
   PRIMARY KEY (`id`),
-  KEY `groups_parent_id_index` (`parent_id`),
-  KEY `index_groups_on_title` (`title`)
+  KEY `groups_parent_id_index` (`parent_id`) USING BTREE,
+  KEY `index_groups_on_title` (`title`) USING BTREE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
 
 CREATE TABLE `groups_roles` (
   `group_id` int(11) NOT NULL DEFAULT '0',
   `role_id` int(11) NOT NULL DEFAULT '0',
   `created_at` datetime DEFAULT NULL,
-  UNIQUE KEY `groups_roles_all_index` (`group_id`,`role_id`),
-  KEY `role_id` (`role_id`),
+  UNIQUE KEY `groups_roles_all_index` (`group_id`,`role_id`) USING BTREE,
+  KEY `role_id` (`role_id`) USING BTREE,
   CONSTRAINT `groups_roles_ibfk_1` FOREIGN KEY (`group_id`) REFERENCES `groups` (`id`),
   CONSTRAINT `groups_roles_ibfk_2` FOREIGN KEY (`role_id`) REFERENCES `roles` (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
 CREATE TABLE `groups_users` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
Actually this is a trivial change - the id column was just moved up from a few lines below.